screens/AboutScreen.js

```diff
@@ -0,0 +1,43 @@
+import { ScrollView, Text } from "react-native";
+import { Card, Avatar, ListItem } from "react-native-elements";
+import { PARTNERS } from "../shared/partners";
+import { useState } from "react";
+
+const Mission = () => {
+
+    return (
+        <Card>
+            <Card.Title>Our Mission</Card.Title>
+            <Card.Divider />
+            <Text style={{ marginBottom: 10 }}>We present a curated database of the best campsites in the vast woods and backcountry of the World Wide Web Wilderness. We increase access to adventure for the public while promoting safe and respectful use of resources. The expert wilderness trekkers on our staff personally verify each campsite to make sure that they are up to our standards. We also present a platform for campers to share reviews on campsites they have visited with each other.</Text>
+        </Card>
+    )
+};
+
+const AboutScreen = () => {
+    const [partners, setPartners] = useState(PARTNERS);
+
+    return (
+        <ScrollView>
+            <Mission />
+            <Card>
+                <Card.Title>Community Partners</Card.Title>
+                {partners.map((partner) => {
+                    return (
+                        <ListItem key={partner.id}>
+                            <Avatar source={partner.image} rounded />
+                            <ListItem.Content>
+                                <ListItem.Title>{partner.description}</ListItem.Title>
```

Comment:
Should this be `{partner.name}`?

Reply:
oops 🙄 Yes you are right! I forgot to check the build carefully.